ui/static/ui/js/listing_resources.jsx

```diff
@@ -239,7 +239,66 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
       }
     });
 
+    var DescriptionListingResource = React.createClass({
+      render: function() {
+        if (this.props.showDetail) {
+          return <div className="tile-blurb">{this.expandDescription()}</div>;
+        }
+
+        return <div className="tile-blurb">{this.wrapDescription()}</div>;
+      },
+      wrapDescription: function() {
+        var resource = this.props.resource;
+        var description = "No description provided.";
+        if (resource.description !== "") {
+          if (resource.description.length > 120) {
+            var wrapDesc ;
+            var positionOfSpace = resource.description.indexOf(" ",  110);
+
+            if (positionOfSpace > 0) {
+              wrapDesc = resource.description.substring(0, positionOfSpace);
+            } else {
+              wrapDesc = resource.description.substring(0, 120);
+            }
+
+            description = <span>
+              {wrapDesc} <br/>
+              <a onClick={this.expandLinkClick}
```

Comment:
Add a `href="#"` to this link and the `Read less` link so that the mouse cursor acts like this is a link. You will also need to adjust `expandLinkClick` and `collapseLinkClick` to take an event and call `event.preventDefault()`